<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivititesManager</class>
 <widget class="QWidget" name="ActivititesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>252</width>
        <height>353</height>
       </rect>
      </property>
      <attribute name="label">
       <string>New Activity</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="nameLbl">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameTxt"/>
       </item>
       <item>
        <widget class="QLabel" name="classifyLbl">
         <property name="text">
          <string>Classification</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="classifyTxt"/>
       </item>
       <item>
        <widget class="QLabel" name="errorMsg">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>252</width>
        <height>353</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Activities</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolBox" name="activityTBx">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="unClassifyPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>234</width>
            <height>307</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Unclassified</string>
          </attribute>
          <layout class="QVBoxLayout" name="unClassifyBox">
           <item>
            <layout class="QGridLayout" name="gridLayout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
